packages/react-scripts-plugin-typescript/src/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const {
+  pushExtensions,
+  pushExclusiveLoader,
+} = require('react-dev-utils/plugins');
+
+function apply(config, { path, paths }) {
```

Comment:
Ah makes sense! Is `react-dev-utils/plugins` a little special then, that you're able to require it just above?